ako-operator/bundle/manifests/ako-operator.clusterserviceversion.yaml

```diff
@@ -29,20 +29,22 @@ metadata:
                 "labelValue": ""
               },
               "servicesAPI": false,
-              "vipPerNamespace": "false"
+              "vipPerNamespace": "false",
+              "primaryInstance": true,
+              "istioEnabled": false,
+              "blockedNamespaceList": [],
+              "ipFamily": ""
```

Comment:
do we have any default value for this or empty string is the default value?

Reply:
Based on AKO Helm chart, empty string is the default value. However, IP family is set to default **V4** in the code internally for all cases (**V4**, **V6**, **empty**). Only in case of cloud type vCenter, ip family if set to **V6**, is used as it is.